### PR TITLE
Update smart contract

### PR DESCRIPTION
**Pull request by anxolin**

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr239--dexreact.review.gnosisdev.com

File: src/hooks/usePlaceOrder.ts [hunk 1]
```diff
@@ -47,23 +47,41 @@ export const usePlaceOrder = (): Result => {
           exchangeApi.getCurrentBatchId(),
         ])
 
-        const validUntil = batchId + DEFAULT_ORDER_DURATION
+        if (sellTokenId !== 0 || buyTokenId !== 0) {
```
Comment (Velenir) [inline src/hooks/usePlaceOrder.ts]: shouldn't this be `&&`?

Comment (anxolin) [reply src/hooks/usePlaceOrder.ts]: totally! good catch!

Comment (alfetopito) [inline src/hooks/usePlaceOrder.ts]: Same comment I have https://github.com/gnosis/dex-react/pull/237 where this change is coming from.

Comment (anxolin) [reply src/hooks/usePlaceOrder.ts]: yep, I've merged that into this branch to use for the demo, I'll remove the commit from the other branch and leave this PR with just the address update

**Pull request merged**